packages/ai/api-report.api.md

```diff
@@ -180,10 +205,10 @@ export interface TldrawAiOptions extends Omit<TldrawAiModuleOptions, 'editor'> {
 }
 
 // @public (undocumented)
-export type TldrawAiPromptOptions = {
-    message: TLAiPrompt['message'];
+export type TldrawAiPromptOptions = (Partial<TLAiPrompt> & {
+    message: string;
     stream?: boolean;
-} | string;
+}) | string;
```

Comment:
Let's not make message required. You might want to send a prompt with just an image, for example